### PR TITLE
Doc updated list of supported webhooks and added example

### PR DESCRIPTION
**Pull request by 8ctopus**
The list of supported webhooks was not up-to-date and I added an example of how to receive the hook.

### CODE REVIEW

File: docs/content/doc/features/webhooks.en-us.md [hunk 1]
```diff
@@ -104,3 +112,64 @@ X-Gitea-Event: push
...
+$decoded = json_decode($payload, true);
+
+// check for json decode errors
+if (json_last_error() !== JSON_ERROR_NONE) {
+    error_log('FAILED - json decode - '. json_last_error());
+    exit();
+}
+
+// check authorization
+if ($decoded['secret'] != $secret_key) {
```
Comment (techknowlogick) [inline docs/content/doc/features/webhooks.en-us.md]: You shouldn't rely on this field existing, it is currently deprecated as we are migrating towards using the checksum header to validate hook content. Would you be able to update this?

Comment (8ctopus) [reply docs/content/doc/features/webhooks.en-us.md]: I didn't know and I actually just learnt about it from you. How does the new system work? Is there any doc about it yet?

Comment (techknowlogick) [reply docs/content/doc/features/webhooks.en-us.md]: Yes, this https://togithub.com/gogs/gogs/issues/4233 and this https://github.com/go-gitea/gitea/issues/3901 should provide details you are looking for :)

Comment (8ctopus) [reply docs/content/doc/features/webhooks.en-us.md]: interesting indeed, let me check that and make changes to the script

Comment (8ctopus) [reply docs/content/doc/features/webhooks.en-us.md]: @techknowlogick I've updated the code snippet and tested it. It works. I hope that's what you had in mind.

**Review by techknowlogick — APPROVED** [review]
Thanks for all the docs improvements you are doing :)

**Review by guillep2k — APPROVED** [review]

**Pull request merged**